ios-trivia-game/API/FirebaseClient.swift

```diff
@@ -315,13 +315,15 @@ class FirebaseClient {
         let newAnswer = ref.child(path).childByAutoId()
         answer.timestamp = String(describing: NSDate())
         newAnswer.setValue(answer.getJson(), withCompletionBlock: { (error, ref) in complete(error, ref) })
+        Logger.instance.log(logLevel: .info, message: "FirebaseClient: Creating \(path)/\(newAnswer.key)")
     }
     
     // Creates a game
     func createGame(gameRoom: NSDictionary, complete: @escaping (Error?, FIRDatabaseReference) -> Void) {
-        let path = "\(Constants.GAME_ROOM_TABLE_NAME)"
-        let newGameRoom = ref.child(path).childByAutoId()
+        let path = "\(Constants.GAME_ROOM_TABLE_NAME)/\(gameRoom["id"]!)"
```

Comment:
my mistake from an initial pr.  We created a unique id but generated another one.
This new code here will use the id passed into createGame